src/core/public/chrome/ui/header/header_breadcrumbs.scss

```diff
@@ -0,0 +1,49 @@
+@import "../../../../public/variables";
+
+@mixin backround($color,$shape) {
+  background-color: #{$color};
+  clip-path: $shape;
+  padding: $euiSizeXS - 2.5 $euiSizeL - $euiSizeXS;
+  border-radius: $euiSizeXS - 2;
+}
+
+.osdHeaderBreadcrumbs {
+  [data-background="osdHeaderBreadcrumbs first"] {
+    @include backround($osdSanFranciscoFog, polygon(0% 0%,100% 0%,calc(100% - #{
+      $euiSizeS + 2}) 100%,0% 100%));
+  }
+
+  [data-background="osdHeaderBreadcrumbs middle"] {
+    @include backround($osdSanFranciscoFog,
+    polygon($euiSizeS + 2 0%,100% 0%,calc(100% - #{$euiSizeS + 2})  100%,0% 100%));
+  }
+
+  [data-background="osdHeaderBreadcrumbs last"] {
+    @include backround($osdPacificSky,polygon(
+      $euiSizeS + 2 0%, 100% 0%, calc(100% - #{$euiSizeS + 2}) 100%, 0% 100%));
+  }
+
+  [data-background="osdHeaderBreadcrumbs first last"] {
+    @include backround($osdPacificSky,polygon(0% 0%,100% 0%,calc(100% - #{$euiSizeS + 2}) 100%,0% 100%));
+  }
+
+  .euiBreadcrumbSeparator {
+    display: none;
+  }
+
+  .euiBreadcrumb:not(.euiBreadcrumb--collapsed) {
+    max-width: 170px;
```

Comment:
Where did we get this size from?

Reply:
so the element has width of 160px by default! we have given some padding as per ux requirement so changing it to 170px.

Reply:
What would happen if we ignore this property? Looking at the code, it looks like the only sideeffect is that the truncation would happen a little earlier, but i could be wrong.

Reply:
Yes thats right...truncation would happen a little earlier! I thought we are adding extra padding so added some width here!

Reply:
If thats the only side effect, i'd rather we not change the value since the truncation accounts for width of the whole bar and not just the word its truncating. And since we aren't changing the truncation logic, just changing this value will not be good if the breadcrumb is too long

Reply:
okay! let me fix that